### ISSUE TITLE
Multifarm not planting Forestry saplings?

### ISSUE THREAD

**Issue by Lykanius** [ForestryMC/ForestryMC#185]
After I built a Forestry multifarm. All was well, it set down the Humus and planted the vanilla saplings I gave it. However, once it cut down THOSE trees, the problem started. It basically refused to plant the Forestry saplings that it harvested from the vanilla trees. I tried a few different Forestry saplings because I had found a few in a village loot chest, and none of them worked. However, any vanilla sapling I tried worked just fine until Forestry converted the saplings to its own type, and then the problem persisted.
tl;dr Why doesn't my Forestry multifarm plant saplings from Forestry, and how can I fix it?
